src/test/java/org/mockitousage/spies/MockCreationShouldNotAffectSpyTest.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockitousage.spies;
-
-import org.junit.Test;
-import org.mockitoutil.TestBase;
-
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-
-public class MockCreationShouldNotAffectSpyTest extends TestBase {
-
-    @Test
-    public void test() {
-        TestClass instance = new TestClass(42);
-
-        TestClass mock = mock(TestClass.class);
-        assertEquals(mock.hashCode(), mock.hashCode());
-        assertEquals(mock, mock);
-
-        TestClass spy = spy(instance);
-        assertEquals(instance.hashCode(), spy.hashCode());
```

Comment:
I guess we can't reinstate the test as-is, but I think we can if we remove this particular line?